app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/welcome/WelcomeScreen.kt

```diff
@@ -121,15 +119,15 @@ private fun WelcomeScreenContent(
         Spacer(modifier = Modifier.weight(1f))
 
         HorizontalPager(state = pagerState) { index ->
-            if (isLandscape) {
-                WelcomeCardLandscape(
+            if (LocalConfiguration.current.isPortrait) {
+                WelcomeCardPortrait(
                     state = state.pages[index],
-                    modifier = Modifier.padding(horizontal = horizontalPadding),
+                    modifier = Modifier.standardHorizontalMargin(landscape = 128.dp),
                 )
             } else {
-                WelcomeCardPortrait(
+                WelcomeCardLandscape(
                     state = state.pages[index],
-                    modifier = Modifier.padding(horizontal = horizontalPadding),
+                    modifier = Modifier.standardHorizontalMargin(landscape = 128.dp),
```

Comment:
Since the landscape margin is the same everywhere on this screen, could we extract it to a parameter `val landscapeHorizontalMargin` or something just to avoid the magic number?